openfisca_web_site/templates/france/fr/tree/demo.mako

```diff
@@ -0,0 +1,245 @@
+        displayData(results);
+        }
+
+
+      function displayData(results){
+        document.getElementById("situationDescription").innerHTML=`
+
+          <li>dispose d'un <strong>revenu disponible</strong> de ` + Math.round(results.revenu_disponible).toLocaleString("fr") + ` €/an, c'est à dire ` + (Math.round(results.revenu_disponible/12)).toLocaleString("fr") + ` €/mois ;</li>
+
+          <li> est <strong>imposé directement</strong> à hauteur de ` + results.taux_imposition + `%( ` + results.impots_directs.toLocaleString("fr") + ` €/an, c'est à dire `+ Math.round(results.impots_directs/12).toLocaleString("fr") + ` €/mois) ;</li>
+
+          <li>perçoit ` + results.prestations_sociales.toLocaleString("fr") + ` €/an de <strong>prestations sociales</strong>, c'est à dire ` + Math.round(results.prestations_sociales/12).toLocaleString("fr") + ` €/mois.</li>`;
+          //Display du graph
+          google.charts.load("current", {"packages":["treemap"]});
+          google.charts.setOnLoadCallback(drawChart);
+
+
+          function drawChart() {
+            var data = google.visualization.arrayToDataTable([
+              ["Element", "Source", "Montant", "Montant (color)"],
+              ["Salaires bruts",null, results.salaires_de_base, results.salaires_de_base],
+              ["Cotisations salariales", "Salaires bruts", results.cotisations_salariales, - results.cotisations_salariales],
+
+              ["Impots directs", "Salaires bruts", results.impots_directs , - results.impots_directs],
+              ["Revenu disponible", "Salaires bruts", results.revenu_disponible, results.revenu_disponible],
+              ["Revenu du travail", "Revenu disponible", results.revenus_du_travail, results.revenus_du_travail],
+              ["Prestations sociales", "Revenu disponible", results.prestations_sociales, results.prestations_sociales],
+            ]);
+
+            tree = new google.visualization.TreeMap(document.getElementById("chart_div"));
+
+            tree.draw(data, {
+              minColor: "#5bc0de",
+              midColor: "#5bc0de",
+              maxColor: "#5cb85c",
+              headerHeight: 55,
+              fontColor: "black",
+              showScale: true
+            });
+
+          }
+          document.getElementById("graph").innerHTML="<div id='chart_div' style='width: 100%; height: 400px;'></div>";
+
+        }
+    </script>
+</%def>
+
+<%def name="h1_content()" filter="trim">
+Calcul du revenu disponible
+</%def>
+
+
+<%def name="page_content()" filter="trim">
+          <p class="lead">Le revenu disponible représente l'ensembles des revenus issus du travail, des pensions, des revenus du capital et des prestations sociales, auxquelles on soustrait les impots directs du ménage.</p>
+
+      <div class="row">
+        <!-- entrée des données -->
+        <div class="col-md-4">
+          <form action="javascript:void(0);">
+            <p><b>Salaire annuel brut du premier conjoint :</b></p>
+            <p>
```

Comment:
You're opening a paragraph without closing it, that's not valid HTML ! 
If you want to add space after a `p`, add a `margin` in CSS.